src/main/java/com/epam/aidial/core/log/GfLogStore.java

```diff
@@ -297,27 +271,24 @@ static String assembleStreamingResponse(Buffer response) {
             result.set("model", model);
 
             if (usage != null) {
+                MergeChunks.removeIndices(usage);
                 result.set("usage", usage);
             }
             if (statistics != null) {
+                MergeChunks.removeIndices(statistics);
                 result.set("statistics", statistics);
             }
             if (systemFingerprint != null) {
                 result.set("system_fingerprint", systemFingerprint);
             }
 
-            if (content.isEmpty()) {
+            if (choices != null) {
+                MergeChunks.removeIndices(choices);
+                result.set("choices", choices);
+            } else {
```

Comment:
Nit: else block is redundant here, the same code as below

Reply:
right fixed.